pybamm/experiment/experiment.py

```diff
@@ -167,12 +167,14 @@ def read_termination(termination):
                         "'80%', '4Ah', or '4A.h'"
                     )
             elif term.endswith("V"):
-                end_discharge_V = term.split("V")[0]
-                termination_dict["voltage"] = (float(end_discharge_V), "V")
+                raise ValueError(
+                    "Voltage termination at the experiment-level is no longer supported. "
+                    "Please set voltage termination conditions at the step level."
+                )
```

Comment:
See other comment, the use case is where the voltage termination is not expected to be hit in every step e.g. GITT https://github.com/pybamm-team/PyBaMM/issues/1826